### PR TITLE
Modern theme toolbar tweaks

### PR DESCRIPTION
**Pull request by Blackbaud-TrevorBurch**

### CODE REVIEW

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-layout?_sp.spa=pr-modern-theme-toolbar-tweaks](https://developer.blackbaud.com/skyux-layout?_sp.spa=pr-modern-theme-toolbar-tweaks)

File: src/app/visual/toolbar/toolbar-visual.component.html [hunk 1]
```diff
@@ -6,6 +6,20 @@
...
 >
   <sky-toolbar>
+    <sky-toolbar-item>
+      <button
+        class="sky-btn sky-btn-primary"
+      >
+        <sky-icon
+          icon="plus-circle"
+          [ngClass]="{
+            'sky-margin-inline-compact' : theme == 'modern'
```
Comment (Blackbaud-AlexKingman) [inline src/app/visual/toolbar/toolbar-visual.component.html:16]: Why does your visual test need to wrap logic around themes? Shouldn't the theme switcher above take care of this for you?

**Review by Blackbaud-AlexKingman — COMMENTED** [review]
One question. Also, is there a story for this work? What are we trying to accomplish?

File: src/app/visual/toolbar/toolbar-visual.component.html (hunk 1 above)
Comment (Blackbaud-TrevorBurch) [reply src/app/visual/toolbar/toolbar-visual.component.html:16]: This is for a small spacing thing that can't be accomplished in-component due to css limitations until we create a button component. I can connect with you directly if you have more questions for this. So the theme switcher changes 99.9% of things but we have to add this class consumer side.

**Review by Blackbaud-AdamFunderburk — APPROVED** [review]
LGTM

File: src/app/visual/toolbar/toolbar-visual.component.html (hunk 1 above)
Comment (Blackbaud-AlexKingman) [reply src/app/visual/toolbar/toolbar-visual.component.html:16]: Hmm.. maybe we should add an inline comment to this then? ...just to help remind ourselves for the visual tests.

Comment (Blackbaud-TrevorBurch) [reply src/app/visual/toolbar/toolbar-visual.component.html:17]: @Blackbaud-AlexKingman added the comment

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-layout?_sp.spa=pr-modern-theme-toolbar-tweaks](https://developer.blackbaud.com/skyux-layout?_sp.spa=pr-modern-theme-toolbar-tweaks)

**Review by Blackbaud-AlexKingman — APPROVED** [review]

**Pull request merged**